File: C++/1010.SimpleCalculate.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(void){
    int codProd, codProd2, uniProd, uniProd2;
    float precoProd, precoProd2, precoTotal;

    cin >> codProd >> uniProd >> precoProd;
    cin >> codProd2 >> uniProd2 >> precoProd2;

    precoTotal = (uniProd*precoProd) + (uniProd2*precoProd2);

    cout << "VALOR A PAGAR: R$ " << fixed << setprecision(2) << precoTotal << endl;
    
    return 0;
}

File: Python/1010.SimpleCalculate.py
codProd, uniProd, precoProd = map(float, input().split())

codProd2, uniProd2, precoProd2 = map(float, input().split())

precoTotal = (uniProd*precoProd) + (uniProd2*precoProd2)

print("VALOR A PAGAR: R$ {:.2f}".format(precoTotal))